AlfrescoApp/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -236,34 +236,34 @@
 "accountdetails.title.newaccount" = "New Account";
 "accountdetails.title.editingaccount" = "Editing Account";
 "accounttype.cloud" = "Alfresco in the Cloud";
-"accounttype.alfrescoServer" = "Alfresco Server";
+"accounttype.alfrescoServer" = "Alfresco One";
 
-"accounttype.footer.alfrescoServer" = "Connect to an Alfresco Server";
-"accounttype.footer.alfrescoCloud" = "Access your Alfresco in the Cloud account";
+"accounttype.footer.alfrescoServer" = "Connect to Alfresco One";
+"accounttype.footer.alfrescoCloud" = "Connect to your Alfresco in the Cloud account";
 
 "accountdetails.placeholder.required" = "Required";
 "accountdetails.placeholder.optional" = "Optional";
 "accountdetails.fields.email" = "Email";
-"accountdetails.placeholder.email" = "example@acme.com";
+"accountdetails.placeholder.email" = "example@alfresco.com";
 "accountdetails.fields.hostname" = "Hostname";
 "accountdetails.fields.description" = "Description";
 "accountdetails.fields.port" = "Port";
 "accountdetails.fields.protocol" = "HTTPS";
 "accountdetails.fields.servicedocument" = "Service Document";
-"accountdetails.fields.syncPreference" = "Sync Files";
-"accountdetails.fields.syncPreference.footer" = "Select to sync favorited content to your device. A warning will be displayed before syncing content over 20 MB";
+"accountdetails.fields.syncPreference" = "Sync Content";
+"accountdetails.fields.syncPreference.footer" = "Select to sync content to your device. A warning will be displayed before syncing content over 20 MB";
```

Comment:
The "Sync on/off" setting can be removed from the account details screen, so this string can be removed. We may need a developer task raised for this.